src/hooks/useColumnWidths.ts

```diff
@@ -131,3 +128,47 @@ function measureColumnWidth(gridRef: React.RefObject<HTMLDivElement | null>, key
   const measuringCell = gridRef.current?.querySelector(selector);
   return measuringCell?.getBoundingClientRect().width;
 }
+
+function getColumnWidthForMeasurement<R, SR>(
+  width: number | string,
+  { minWidth, maxWidth }: CalculatedColumn<R, SR>
+) {
+  const widthWithUnit = typeof width === 'number' ? `${width}px` : width;
+
+  // don't break in Node.js (SSR) and jsdom
+  if (typeof CSS === 'undefined') {
+    return widthWithUnit;
+  }
+
+  const hasMaxWidth = maxWidth != null;
```

Comment:
Do we need to handle `maxWidth` here if we set it on measuring cells?

Reply:
Not 100% sure. My understanding is that grid computes the column widths and then apply min/max style from the measuring cell. This is the reason there was some left over space in the bug you found. I will check again

Reply:
Also, resizing does not work properly without this as `clampColumnWidth` helper was removed

Reply:
Found another edge case.